src/main/java/com/endava/utility/Main.java

```diff
@@ -0,0 +1,24 @@
+package com.endava.utility;
+
+import com.endava.utility.list.Lists;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        List<String> listOfStrings = Lists.createList("jedan","dva","tri");
+        List<Integer> listOfIntegers = Lists.createList(1, 2, 3);
+        printList(listOfStrings);
+        printList(listOfIntegers);
+
+        List<String> listOfStrings2 = com.google.common.collect.Lists.newArrayList("jedan","dva","tri");
+        printList(listOfStrings2);
+    }
+
+    private static <E> void printList(List<E> list){
```

Comment:
Is this method different than simply calling  `listOfStrings.toString()`

Reply:
I have removed that as well because there is not difference.